src/DaySlot.jsx

```diff
@@ -150,11 +170,13 @@ let DaySlot = React.createClass({
         <div
           key={'evt_' + idx}
           style={{...xStyle, ...style}}
-          title={label + ': ' + title }
+          title={(typeof label ? label + ': ' : '') + title }
```

Comment:
whats going on with this `typeof` here? Also is there a way in which this could not exist?